x/shield/simulation/operations.go

```diff
@@ -475,6 +475,9 @@ func ProposalContents(k keeper.Keeper, sk types.StakingKeeper) []simulation.Weig
 // SimulateShieldClaimProposalContent generates random shield claim proposal content
 func SimulateShieldClaimProposalContent(k keeper.Keeper, sk types.StakingKeeper) simulation.ContentSimulatorFn {
 	return func(r *rand.Rand, ctx sdk.Context, accs []simulation.Account) govtypes.Content {
+		if ctx.BlockHeight() < common.Update1Height {
+			return nil
+		}
```

Comment:
I don't think we need this in simulation since there isn't an incompatible history sim needs to replay?

Reply:
But please correct me if I'm wrong

Reply:
it'll fail in the handler so we do need it lol

Reply:
We checks in handlers, so it will return error if we don't add it here.

Reply:
Ah I understand now, thanks guys! 
But will simulation ever reach the update height?

Reply:
It is set to 1 for now. Just in case we set it to some number that could be caught by simulations in the future.